components/onboarding/.env.example

```
@@ -117,4 +117,6 @@ MAX_PAGINATION_MONTH_DATE_RANGE=3
 
 # TRANSACTION GRPC
 TRANSACTION_GRPC_ADDRESS=midaz-transaction
-TRANSACTION_GRPC_PORT=3011
\ No newline at end of file
+TRANSACTION_GRPC_PORT=3011
+# Optional: Timeout for gRPC health check before creating accounts/assets (default: 5s)
+# GRPC_HEALTH_CHECK_TIMEOUT=5s
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)